src/web/app/pages-instructor/instructor-session-edit-page/copy-questions-from-other-sessions-modal/__snapshots__/copy-questions-from-other-sessions-modal.component.spec.ts.snap

```diff
@@ -145,7 +145,7 @@ exports[`CopyQuestionsFromOtherSessionsModalComponent should snap when feedback
                     <tr>
                       <td>
                         <input
-                          class="form-control ng-untouched ng-pristine ng-valid"
+                          ng-untouched ng-pristine ng-valid"
```

Comment:
looks good, just a reminder to update the snapshot as well when you remove the `form-controller` class

Reply:
I have removed it, sir.

Reply:
ah my bad I didn't realise the code was updated already, all looks good! 

for future PRs do re-request reviews from us when the changes have been made, preferably when the tests complete :)

Reply:
Sure, noted that down :)